sdk/include/psdk/versionhelpers.h

```diff
@@ -43,93 +43,111 @@ IsWindowsVersionOrGreater(WORD wMajorVersion, WORD wMinorVersion, WORD wServiceP
 }
 
 VERSIONHELPERAPI
-IsWindowsXPOrGreater()
+IsWindowsXPOrGreater(void)
 {
     return IsWindowsVersionOrGreater(5, 1, 0);
 }
 
 VERSIONHELPERAPI
-IsWindowsXPSP1OrGreater()
+IsWindowsXPSP1OrGreater(void)
 {
     return IsWindowsVersionOrGreater(5, 1, 1);
 }
 
 VERSIONHELPERAPI
-IsWindowsXPSP2OrGreater()
+IsWindowsXPSP2OrGreater(void)
 {
     return IsWindowsVersionOrGreater(5, 1, 2);
 }
 
 VERSIONHELPERAPI
-IsWindowsXPSP3OrGreater()
+IsWindowsXPSP3OrGreater(void)
 {
     return IsWindowsVersionOrGreater(5, 1, 3);
 }
 
 VERSIONHELPERAPI
-IsWindowsVistaOrGreater()
+IsWindowsServer2003OrGreater(void)
```

Comment:
This is a PSDK header. MS's PSDK does not have this, so neither should ours.

Reply:
Right. What would you suggest?
* Add `#ifdef __REACTOS__`, as `IsReactOS()` has.
* "Duplicate" needed code to somewhere else.
* Use "XP+ && Server && !Vista+" instead, which should probably be good enough.
* ...

Reply:
Wrapping code with `__REACTOS__`, in my humble opinion, could work for this case. That is, if everyone agrees with that of course.